src/main/java/seedu/address/ui/Ui.java

```diff
@@ -10,4 +10,6 @@ public interface Ui {
     /** Starts the UI (and the App).  */
     void start(Stage primaryStage);
 
+    void showInvalidFormatInJsonFile();
```

Comment:
I think the naming of the function can be changed to "displayInvalidJsonFileMessage()" to improve readability. Otherwise, I would expect this function to point out where goes wrong with the JSON file in format.

Reply:
that's a good point ! i will change it